src/mavsdk/plugins/component_metadata/component_metadata.cpp

```diff
@@ -0,0 +1,36 @@
+// WARNING: THIS FILE IS AUTOGENERATED! As such, it should not be edited.
+// Edits need to be made to the proto files
+// (see
+// https://github.com/mavlink/MAVSDK-Proto/blob/master/protos/component_metadata/component_metadata.proto)
```

Comment:
Can you also make the pull request to MAVSDK-Proto?

Reply:
Yes: https://github.com/mavlink/MAVSDK-Proto/pull/338

Reply:
Thanks!